src/main/java/pl/javastart/task/PersonBuilder.java

```diff
@@ -0,0 +1,44 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class PersonBuilder {
+    private Scanner scanner = new Scanner(System.in);
+
+    private Person createPerson() {
+        System.out.print("Podaj imię: ");
+        String firstname = scanner.nextLine();
+        System.out.print("Podaj wiek: ");
+        int age = scanner.nextInt();
+        scanner.nextLine();
+        return new Person(firstname, age);
+    }
+
+    Person[] createPersonArray(int size) {
+        int i = 0;
+        Person[] persons = new Person[size];
+        do {
+            Person person = createPerson();
+            if (isUnique(person, persons, i)) {
+                System.out.println("Duplikat");
+                persons[i] = createPerson();
```

Comment:
Nazwa tej metody nie pasuje do tego, co tutaj robisz.
W tym momencie jest "jeżeli obiekt jest unikalny, to wyświetl, że jest duplikatem". Powinno być raczej "jeżeli obiekt jest duplikatem, to wyświetl duplikat", albo "jeżeli obiekt NIE jest unikalny, to wyświetl duplikat".
Także albo zmień nazwę metody z isUnique na coś innego, albo odwróć tutaj warunek i bloki if-else miejscami